Jenkinsfile

```diff
@@ -72,7 +72,14 @@ node('ca-jenkins-agent') {
 
     // Build the application
     pipeline.build(
-        timeout: [time: 5, unit: 'MINUTES']
+        operation: {
+            sh "npm run build"
+            sh "npm run checkTestsCompile"
```

Comment:
If we decide to make `checkTestCOmpile` part of the `postbuild` npm script, then we could remove this `operation`